src/layout/components/MainContainer.tsx

```diff
@@ -1,7 +1,17 @@
+import React from 'react';
 import styled from 'styled-components';
+import { Container } from '@material-ui/core';
 
 /* Main container with background img for all pages */
-export const MainContainer = styled('div')`
-    background: url('${process.env.PUBLIC_URL}/img/background-img.png') center top / cover no-repeat;
-    min-height: 100vh;
+const BackgroundContainer = styled('div')`
+  background: url('${process.env.PUBLIC_URL}/img/background-img.png') center top / cover no-repeat;
+  min-height: 100vh;
 `;
+
+export const MainContainer: React.FC<{}> = ({ children }) => (
+  <BackgroundContainer>
+    <Container maxWidth="md">
+      { children! }
```

Comment:
It solves issue described here: https://github.com/mui-org/material-ui/issues/21711

Reply:
thx a lot :)